conda/gateways/repodata/jlap/fetch.py

```diff
@@ -358,9 +358,9 @@ def request_url_jlap_state(
             state.set_has_format("jlap", True)
             need_jlap = False
         except ValueError:
-            log.info("Checksum not OK")
-        except IndexError as e:
-            log.info("Incomplete file?", exc_info=e)
+            log.info("Checksum not OK on JLAP range request. Retry with complete JLAP.")
```

Comment:
This error will happen if the remote file rolls over but our local position is still within the range of the new remote file, that is the reason the format has a trailing checksum.

Reply:
My own `pytest --cov` runs these lines that Codecov says did not run.